services/QuillLMS/db/structure.sql

```diff
@@ -44,6 +44,20 @@ CREATE EXTENSION IF NOT EXISTS hstore WITH SCHEMA public;
 COMMENT ON EXTENSION hstore IS 'data type for storing sets of (key, value) pairs';
```

Comment:
It's not clear to me what this large diff has to do with the commit.  Is this just pulling in stuff that should have been pulled in other commits?

Reply:
This file accidentally made it into the commit. I removed it.